src/constants/translations/en/my-resources-page.json

```diff
@@ -22,7 +22,8 @@
     "file": "File",
     "size": "Size",
     "lastUpdate": "Last update",
-    "emptyAttachments": "You have no attachments yet"
+    "emptyAttachments": "You have no attachments yet",
+    "validationError": "File name cannot be longer than 55 symbol."
```

Comment:
I guess you can use `File name cannot be longer than {{number}} symbol.`.
It is necessary to pass amount of symbols. As for me, it is reusable and we change number in one place.